packages/sdk/test/integration/client_test.ts

```diff
@@ -852,7 +852,7 @@ describe.sequential('Client', function () {
       await c2.sync();
 
       // 01. c1 increases the counter for creating snapshot.
-      for (let i = 0; i < 500; i++) {
+      for (let i = 0; i < 1000; i++) {
         d1.update((r) => r.counter.increase(1));
       }
```

Comment:
I changed this value because the `DefaultSnapshotThreshold` is changed in [this PR](https://github.com/yorkie-team/yorkie/pull/1051/files#diff-a52f762042c4d9716c269b541193616c0deeb6ec999ede6e5f41271df6e00f53.).